src/languageclient.rs

```diff
@@ -953,6 +953,8 @@ impl State {
                     let loc = arr.get(0).ok_or_else(|| err_msg("Not found!"))?;
                     self.edit(&goto_cmd, loc.uri.filepath()?)?;
                     self.cursor(loc.range.start.line + 1, loc.range.start.character + 1)?;
+                    let cur_file: String = self.eval("expand('%')")?;
+                    self.echomsg_ellipsis(format!("[LC]: {} {}:{}", cur_file, loc.range.start.line + 1, loc.range.start.character + 1))?;
```

Comment:
GotoDefinition doesn't necessarily go the same file. This echoed filename should be the target file, right?

Reply:
OK. Then it's not necessary to use `eval` to get the filename. It's better to get that from `loc` variable. Every `eval` is an rpc call and thus has larger overhead.

Reply:
Here is my revision:

```rust
self.echomsg_ellipsis(format!("[LC]: {} {}:{}", loc.uri.filepath().unwrap().display(), loc.range.start.line + 1, loc.range.start.character + 1))?;
```

However, it always shows the absolute path:

```
[LC]: /Users/xlc/.vim/plugged/LanguageClient-neovim/src/vim.rs 214:12
```

whereas I hope when I'm still inside the project, it's better to show the relative path:

```
[LC]: src/vim.rs 214:12
```

Is there another solution?

Reply:
I see. Make sense.